app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorModel.kt

```diff
@@ -23,4 +24,10 @@ data class SyncIndicatorModel(
   fun syncIndicatorStateChanged(newState: SyncIndicatorState): SyncIndicatorModel {
     return copy(syncIndicatorState = newState)
   }
+
+  fun syncIndicatorSyncedTimerIncremented(newTime: Long): SyncIndicatorModel {
+    val syncedState = syncIndicatorState as Synced
```

Comment:
Is there a possibility that this cast can fail?

Reply:
Yes. I will add a check to only trigger this in `Synced` state.